src/mysql.erl

```diff
@@ -175,32 +181,86 @@ start_link(Options) ->
          Query :: iodata(),
          Result :: query_result().
 query(Conn, Query) ->
-    query_call(Conn, {query, Query}).
+    query_call(Conn, {query, Query, undefined, undefined}).
 
 %% @doc Depending on the 3rd argument this function does different things.
 %%
-%% If the 3rd argument is a list, it executes a parameterized query. This is
-%% equivallent to query/4 with the query timeout as given to start_link/1.
+%% If the 3rd argument is a list, it executes a parameterized query and applies
+%% no filtering/mapping of the result rows. This is equivallent to query/5
+%% with undefined as filter/map function and the query timeout as given to
```

Comment:
> not saying I can do it *well*, explaining things is one of my weak points, especially so since English is not my native language

That's not true :-)